diff --git a/node_modules/firebase/auth/dist/esm/index.esm.js b/node_modules/firebase/auth/dist/esm/index.esm.js
index 8f2de44..93452c1 100644
--- a/node_modules/firebase/auth/dist/esm/index.esm.js
+++ b/node_modules/firebase/auth/dist/esm/index.esm.js
@@ -1,2 +1,2 @@
-export * from '@firebase/auth';
+export * from '@firebase/auth/dist/esm/index.esm.js';
 //# sourceMappingURL=index.esm.js.map
diff --git a/node_modules/firebase/firestore/dist/esm/index.esm.js b/node_modules/firebase/firestore/dist/esm/index.esm.js
index 92c42b8..7b59362 100644
--- a/node_modules/firebase/firestore/dist/esm/index.esm.js
+++ b/node_modules/firebase/firestore/dist/esm/index.esm.js
@@ -1,2 +1,2 @@
-export * from '@firebase/firestore';
+export * from '@firebase/firestore/dist/esm/index.esm.js';
 //# sourceMappingURL=index.esm.js.map
